src/index.ts

```diff
@@ -41,17 +50,40 @@ async function start() {
         registerPublicHandlers(SERVICE_NAME, inventoryPublicHandlers);
 
         process.on("SIGINT", () => {
-            console.log("[ITEM-STORE] Gracefully shutting down...");
+            logModel.date = new Date();
```

Comment:
is there a way to configure the logger so that we don't have to specify the date every time? 
I believe if I call logger.log(), the date should be automatically added for me